packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -224,7 +222,6 @@ const TESTS = [
       ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.subscriptionFirstInvoice }],
     ])],
     ['html', [
-      { test: 'include', expected: configHref('subscriptionPrivacyUrl', 'subscription-first-invoice', 'subscription-privacy') },
```

Comment:
nit: Could check for new privacy url?

Reply:
heh yeah I skipped it because everything's hard coded; I'll add a check.  👍